app/scripts/store/dimStoreItems.directive.js

```diff
@@ -224,6 +224,15 @@
       vm.data = _.groupBy(vm.store.items, function(item) {
         return item.type;
       });
+
+      if(_.where(vm.store.items, {type: 'Lost Items'}).length >= 20) {
+        dimInfoService.show('lostitems', {
+          type: 'warning',
+          title: 'Postmaster Limit',
+          body: 'There are 20 lost items at the Postmaster on the ' + vm.store.name + '. Any new items will overwrite the existing.',
```

Comment:
Mind saying "your" instead of "the"? As in "your male exo hunter". It's how we do it elsewhere.